sonar-db/src/main/java/org/sonar/db/version/AlterColumnsTypeBuilder.java

```diff
@@ -70,19 +74,19 @@ public AlterColumnsBuilder updateColumn(ColumnDef columnDef) {
 
   private List<String> createPostgresQuery() {
     StringBuilder sql = new StringBuilder("ALTER TABLE " + tableName + " ");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ALTER TABLE " 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1192)